packages/devtools_app/lib/src/screens/debugger/debugger_controller.dart

```diff
@@ -31,25 +27,20 @@ final _log = DebugTimingLogger('debugger', mute: true);
 
 /// Responsible for managing the debug state of the app.
 class DebuggerController extends DisposableController
-    with AutoDisposeControllerMixin, SearchControllerMixin<SourceToken> {
+    with AutoDisposeControllerMixin {
   // `initialSwitchToIsolate` can be set to false for tests to skip the logic
   // in `switchToIsolate`.
   DebuggerController({this.initialSwitchToIsolate = true}) {
-    programExplorerController = ProgramExplorerController();
     autoDisposeStreamSubscription(
       serviceManager.onConnectionAvailable.listen(_handleConnectionAvailable),
     );
     if (serviceManager.hasService) {
       initialize();
     }
-    _scriptHistoryListener = () {
-      final currentScriptValue = scriptsHistory.current.value;
-      if (currentScriptValue != null)
-        _showScriptLocation(ScriptLocation(currentScriptValue));
-    };
-    scriptsHistory.current.addListener(_scriptHistoryListener);
   }
 
+  final debuggerCodeViewController = CodeViewController();
```

Comment:
nit: maybe just CodeViewController? DebuggerController.debuggerCodeViewController is a bit redundant